model-archiver/README.md

```diff
@@ -140,13 +137,17 @@ A serialized file (.pt or .pth) should be a checkpoint in case of torchscript an
 
 ### Handler
 
-Handler can be TorchServe's inbuilt handler name or path to a py to handle custom TorchServe inference logic. TorchServe supports following handlers out or box:
+Handler can be TorchServe's inbuilt handler name or path to a py file to handle custom TorchServe inference logic. TorchServe supports following handlers out or box:
 1. `image_classifier`
 2. `object_detector`
 3. `text_classifier`
 4. `image_segmenter`
 
-For more details refer [default handler documentation](../docs/default_handlers.md)
+In case of custom handler, if you plan to provide just `module_name` or `module_name:entry_point_function_name` then make sure that it is prefixed with absolute or relative path of python file.
+e.g. if your custom handler custom_image_classifier.py is in /home/serve/examples then
+`--handler /home/serve/examples/custom_image_classifier` or if it has my_entry_point module level function then `--handler /home/serve/examples/custom_image_classifier:my_entry_point_func`
```

Comment:
@dhaniram-kshirsagar Shouldn't this be --handler /home/serve/examples/custom_image_classifier**.py**? Where is the module_name getting set in case of Custom Handlers?

Reply:
The framework had support for custom entry_point_function_names i.e. you can name entrypoint function as handle_request instead of the standard `handle` in custom handlers. However, due to bug which is fixed in this PR, this feature was hidden from the user [especially for users coming from MMS #465 ].  With this, we assume that the custom handler module_name or py file has the same name. The handler name or handler name with entry_point_func [i.e. handler:func_name] is recorded in mar Manifest file.

This `--handler /home/serve/examples/custom_image_classifier` is available as an additional option, along with main use case of supplying entry_point_function name with handler.